<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1196</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>1161</width>
      <height>701</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
/* Style des boutons à l'intérieur des onglets */
QPushButton {
    background-color: #007bff; /* Fond bleu vif pour les boutons */
    color: white; /* Texte en blanc */
    border: 1px solid #0056b3; /* Bordure bleue foncée pour les boutons */
    border-radius: 8px; /* Coins arrondis pour les boutons */
    padding: 10px 20px; /* Espacement autour du texte du bouton */
    font-size: 16px; /* Taille du texte du bouton */
    transition: background-color 0.3s, border-color 0.3s, transform 0.3s; /* Effets de transition */
}

QPushButton:hover {
    background-color: #0069d9; /* Fond bleu clair au survol */
    border-color: #004085; /* Bordure plus foncée au survol */
    transform: scale(1.05); /* Effet d'agrandissement léger au survol */
}

QPushButton:pressed {
    background-color: #0056b3; /* Fond plus foncé lorsque le bouton est pressé */
    border-color: #003366; /* Bordure encore plus foncée quand pressé */
}
/* Style général du QTabWidget */
QTabWidget::pane {
    background-color: #add8e6; /* Bleu clair pour l'arrière-plan */
    border: 1px solid #007bff; /* Optionnel : bordure bleue autour du QTabWidget */
}

/* Style pour les onglets dans la barre de tabulation (QTabBar) */
QTabBar::tab {
    background-color: #f0f0f0; /* Fond gris clair pour les onglets inactifs */
    color: #000000; /* Texte noir */
    border: 1px solid #ccc; /* Bordure grise autour des onglets */
    padding: 8px 16px;
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
}

/* Style pour l'onglet sélectionné */
QTabBar::tab:selected {
    background-color: #ffffff; /* Fond blanc pour l'onglet actif */
    color: #000000; /* Texte noir pour l'onglet actif */
}

/* Style pour l'onglet au survol */
QTabBar::tab:hover {
    background-color: #e0e0e0; /* Fond gris plus clair au survol */
}
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="facture">
     <attribute name="title">
      <string>facture</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>251</width>
        <height>431</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: #F0F0F0;    /* Light gray background */
    color: #333333;               /* Dark text color */
    border: 2px solid #0078D7;    /* Blue contour border */
    border-radius: 10px;          /* Rounded corners */
    padding: 5px;                 /* Padding inside the label */
    font-size: 16px;              /* Font size */
    font-weight: bold;            /* Bold text */
    text-align: center;           /* Center the text */
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>210</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* Style pour QLineEdit */
QLineEdit {
    background-color: #f8f8f0; /* Blanc cassé */
    color: #333; /* Texte gris foncé */
    border: 2px solid #007bff; /* Bordure bleu */
    border-radius: 5px; /* Coins arrondis */
    padding: 8px; /* Espacement interne */
    font-size: 14px; /* Taille de la police */
    transition: border-color 0.3s, box-shadow 0.3s; /* Effet fluide pour le survol */
}

/* Style pour QLineEdit lorsqu'il est survolé */
QLineEdit:hover {
    border-color: #0056b3; /* Bordure bleu foncé au survol */
    box-shadow: 0 0 8px #0056b3; /* Ombre bleue autour de l'élément au survol */
}

/* Style pour QLineEdit lorsqu'il est en focus */
QLineEdit:focus {
    border-color: #00bfff; /* Bordure bleu ciel lorsqu'il est sélectionné */
    box-shadow: 0 0 12px #00bfff; /* Ombre plus intense lorsqu'il est sélectionné */
}
</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>132</x>
        <y>150</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* Style pour QLineEdit */
QLineEdit {
    background-color: #f8f8f0; /* Blanc cassé */
    color: #333; /* Texte gris foncé */
    border: 2px solid #007bff; /* Bordure bleu */
    border-radius: 5px; /* Coins arrondis */
    padding: 8px; /* Espacement interne */
    font-size: 14px; /* Taille de la police */
    transition: border-color 0.3s, box-shadow 0.3s; /* Effet fluide pour le survol */
}

/* Style pour QLineEdit lorsqu'il est survolé */
QLineEdit:hover {
    border-color: #0056b3; /* Bordure bleu foncé au survol */
    box-shadow: 0 0 8px #0056b3; /* Ombre bleue autour de l'élément au survol */
}

/* Style pour QLineEdit lorsqu'il est en focus */
QLineEdit:focus {
    border-color: #00bfff; /* Bordure bleu ciel lorsqu'il est sélectionné */
    box-shadow: 0 0 12px #00bfff; /* Ombre plus intense lorsqu'il est sélectionné */
}
</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>132</x>
        <y>90</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* Style pour QLineEdit */
QLineEdit {
    background-color: #f8f8f0; /* Blanc cassé */
    color: #333; /* Texte gris foncé */
    border: 2px solid #007bff; /* Bordure bleu */
    border-radius: 5px; /* Coins arrondis */
    padding: 8px; /* Espacement interne */
    font-size: 14px; /* Taille de la police */
    transition: border-color 0.3s, box-shadow 0.3s; /* Effet fluide pour le survol */
}

/* Style pour QLineEdit lorsqu'il est survolé */
QLineEdit:hover {
    border-color: #0056b3; /* Bordure bleu foncé au survol */
    box-shadow: 0 0 8px #0056b3; /* Ombre bleue autour de l'élément au survol */
}

/* Style pour QLineEdit lorsqu'il est en focus */
QLineEdit:focus {
    border-color: #00bfff; /* Bordure bleu ciel lorsqu'il est sélectionné */
    box-shadow: 0 0 12px #00bfff; /* Ombre plus intense lorsqu'il est sélectionné */
}
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    color: #000000; /* Texte noir */
    font-weight: bold; /* Texte en gras */
    font-size: 25px; /* Taille de la police à 35 */
}
</string>
      </property>
      <property name="text">
       <string>ref</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>510</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supprimer">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>420</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color=rgb(0, 0, 255);
alternate-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.33 rgba(0, 0, 0, 255), stop:0.34 rgba(255, 30, 30, 255), stop:0.66 rgba(255, 0, 0, 255), stop:0.67 rgba(255, 255, 0, 255), stop:1 rgba(255, 255, 0, 255));</string>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifierButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>420</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    color: #000000; /* Texte noir */
    font-weight: bold; /* Texte en gras */
    font-size: 25px; /* Taille de la police à 35 */
}
</string>
      </property>
      <property name="text">
       <string>nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    color: #000000; /* Texte noir */
    font-weight: bold; /* Texte en gras */
    font-size: 25px; /* Taille de la police à 35 */
}
</string>
      </property>
      <property name="text">
       <string>mode</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>340</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    color: #000000; /* Texte noir */
    font-weight: bold; /* Texte en gras */
    font-size: 25px; /* Taille de la police à 35 */
}
</string>
      </property>
      <property name="text">
       <string>prix</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>290</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* Style pour QDateEdit */
QDateEdit {
    background-color: #f8f8f0; /* Blanc cassé */
    color: #000000; /* Texte noir */
    border: 2px solid #007bff; /* Bordure bleue */
    border-radius: 5px; /* Coins arrondis */
    padding: 6px; /* Espacement interne */
    font-size: 14px; /* Taille de la police */
    min-width: 120px; /* Largeur minimale pour le champ de date */
}

/* Style au survol de QDateEdit */
QDateEdit:hover {
    border-color: #0056b3; /* Bordure bleu foncé au survol */
}

/* Style pour QDateEdit lorsqu'il est en focus */
QDateEdit:focus {
    border-color: #00bfff; /* Bordure bleu ciel au focus */
    box-shadow: 0 0 8px #00bfff; /* Ombre bleue au focus */
}

/* Style pour la flèche de descente du QDateEdit */
QDateEdit::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left: 1px solid #007bff; /* Séparation entre le champ et le bouton de descente */
    background-color: #007bff; /* Fond bleu pour le bouton */
    border-top-right-radius: 5px; /* Coins arrondis en haut à droite */
    border-bottom-right-radius: 5px; /* Coins arrondis en bas à droite */
}

/* Style pour l'icône de la flèche de descente */
QDateEdit::down-arrow {
    image: url(:/images/down-arrow.png); /* Remplacer par l'icône personnalisée de flèche vers le bas */
    width: 12px; /* Largeur de l'icône */
    height: 12px; /* Hauteur de l'icône */
}

/* Optionnel: Style au survol du bouton de descente */
QDateEdit::drop-down:hover {
    background-color: #0056b3; /* Changement de couleur au survol */
}

/* Optionnel: Style des boutons fléchés dans l'affichage du calendrier */
QCalendarWidget QToolButton {
    color: #ffffff; /* Couleur du texte des boutons du calendrier */
    background-color: #007bff; /* Fond bleu pour les boutons du calendrier */
    border: none; /* Pas de bordure pour les boutons */
    border-radius: 4px; /* Coins légèrement arrondis */
    padding: 5px;
}

/* Style pour les boutons fléchés (mois précédent/suivant) */
QCalendarWidget QToolButton::hover {
    background-color: #0056b3; /* Fond bleu foncé au survol */
}

/* Style des numéros dans le calendrier */
QCalendarWidget QAbstractItemView {
    background-color: #ffffff; /* Fond blanc pour les jours */
    selection-background-color: #007bff; /* Fond bleu pour le jour sélectionné */
    selection-color: white; /* Texte blanc pour le jour sélectionné */
    color: #000000; /* Texte noir pour les jours non sélectionnés */
}
</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>70</y>
        <width>731</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView {
    background-color: #add8e6; /* Arrière-plan bleu clair */
    alternate-background-color: #e0f7fa; /* Couleur alternative pour les lignes */
    border: 1px solid #007bff; /* Bordure bleu vif */
    gridline-color: #007bff; /* Lignes de grille en bleu */
    selection-background-color: #87cefa; /* Bleu clair pour la sélection */
    selection-color: #ffffff; /* Texte blanc pour les cellules sélectionnées */
    font-size: 14px; /* Taille de police par défaut */
}

QTableView::item {
    background-color: transparent; /* Fond transparent pour les éléments */
    color: #000000; /* Texte noir */
}

QHeaderView::section {
    background-color: #007bff; /* Fond bleu vif pour l'en-tête */
    color: white; /* Texte blanc dans les en-têtes */
    padding: 4px;
    border: 1px solid #0056b3; /* Bordure pour les en-têtes */
}



</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_generatePDF">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>420</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>pdf</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_search">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>420</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>chercher</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_search">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>420</y>
        <width>113</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* Style pour QLineEdit */
QLineEdit {
    background-color: #f8f8f0; /* Blanc cassé */
    color: #333; /* Texte gris foncé */
    border: 2px solid #007bff; /* Bordure bleu */
    border-radius: 5px; /* Coins arrondis */
    padding: 8px; /* Espacement interne */
    font-size: 14px; /* Taille de la police */
    transition: border-color 0.3s, box-shadow 0.3s; /* Effet fluide pour le survol */
}

/* Style pour QLineEdit lorsqu'il est survolé */
QLineEdit:hover {
    border-color: #0056b3; /* Bordure bleu foncé au survol */
    box-shadow: 0 0 8px #0056b3; /* Ombre bleue autour de l'élément au survol */
}

/* Style pour QLineEdit lorsqu'il est en focus */
QLineEdit:focus {
    border-color: #00bfff; /* Bordure bleu ciel lorsqu'il est sélectionné */
    box-shadow: 0 0 12px #00bfff; /* Ombre plus intense lorsqu'il est sélectionné */
}
</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboboxSort">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>30</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* Style pour QComboBox */
QComboBox {
    background-color: #f8f8f0; /* Blanc cassé */
    color: #000000; /* Texte noir */
    border: 2px solid #007bff; /* Bordure bleu */
    border-radius: 5px; /* Coins arrondis */
    padding: 6px; /* Espacement interne */
    font-size: 14px; /* Taille de la police */
    combobox-popup: 0; /* Affichage standard du menu déroulant */
}

/* Style pour les éléments dans le menu déroulant */
QComboBox QAbstractItemView {
    background-color: #f8f8f0; /* Couleur du menu déroulant */
    border: 2px solid #007bff; /* Bordure bleue autour du menu déroulant */
    selection-background-color: #0056b3; /* Fond bleu foncé lors de la sélection */
    selection-color: white; /* Texte blanc pour l'élément sélectionné */
    color: #000000; /* Texte noir pour les éléments non sélectionnés */
}

/* Style pour QComboBox lorsqu'il est survolé */
QComboBox:hover {
    border-color: #0056b3; /* Bordure bleu foncé au survol */
}

/* Style pour QComboBox lorsqu'il est en focus */
QComboBox:focus {
    border-color: #00bfff; /* Bordure bleu ciel lorsqu'il est en focus */
    box-shadow: 0 0 8px #00bfff; /* Ombre bleue au focus */
}

/* Style pour la flèche déroulante de QComboBox */
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left: 1px solid #007bff; /* Bordure à gauche de la flèche */
    background-color: #007bff; /* Fond bleu pour la flèche */
}

/* Icône de la flèche dans le QComboBox */
QComboBox::down-arrow {
    image: url(:/images/down-arrow.png); /* Remplacer par l'icône de la flèche */
    width: 12px;
    height: 12px;
}

</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    color: #000000; /* Texte noir */
    font-weight: bold; /* Texte en gras */
    font-size: 25px; /* Taille de la police à 35 */
}
</string>
      </property>
      <property name="text">
       <string>date</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>350</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* Style pour QDoubleSpinBox */
QDoubleSpinBox {
    background-color: #f8f8f0; /* Blanc cassé */
    color: #000000; /* Texte noir */
    border: 2px solid #007bff; /* Bordure bleue */
    border-radius: 5px; /* Coins arrondis */
    padding: 6px; /* Espacement interne */
    font-size: 14px; /* Taille de la police */
    min-width: 100px; /* Largeur minimale */
}

/* Style au survol de QDoubleSpinBox */
QDoubleSpinBox:hover {
    border-color: #0056b3; /* Bordure bleu foncé au survol */
}

/* Style pour QDoubleSpinBox lorsqu'il est en focus */
QDoubleSpinBox:focus {
    border-color: #00bfff; /* Bordure bleu ciel au focus */
    box-shadow: 0 0 8px #00bfff; /* Ombre bleue au focus */
}

/* Style pour les boutons fléchés de QDoubleSpinBox */
QDoubleSpinBox::up-button, QDoubleSpinBox::down-button {
    background-color: #007bff; /* Fond bleu pour les boutons de flèches */
    border: none; /* Pas de bordure */
    width: 16px; /* Largeur des boutons */
    height: 16px; /* Hauteur des boutons */
    border-radius: 3px; /* Coins légèrement arrondis */
}

/* Changement de couleur des boutons au survol */
QDoubleSpinBox::up-button:hover, QDoubleSpinBox::down-button:hover {
    background-color: #0056b3; /* Bleu foncé au survol */
}

/* Style de la flèche haut */
QDoubleSpinBox::up-arrow {
    image: url(:/images/up-arrow.png); /* Remplacer par votre icône de flèche vers le haut */
    width: 10px;
    height: 10px;
}

/* Style de la flèche bas */
QDoubleSpinBox::down-arrow {
    image: url(:/images/down-arrow.png); /* Remplacer par votre icône de flèche vers le bas */
    width: 10px;
    height: 10px;
}

/* Style pour le bouton pressé */
QDoubleSpinBox::up-button:pressed, QDoubleSpinBox::down-button:pressed {
    background-color: #0056b3; /* Fond plus sombre lorsque le bouton est pressé */
}
</string>
      </property>
     </widget>
     <widget class="QPushButton" name="statButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>480</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>statstique(mode)</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="calucleextras">
     <attribute name="title">
      <string>calcule extras</string>
     </attribute>
     <widget class="QTableView" name="tableView2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>321</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_afficher">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>380</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>afficher</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView3">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>90</y>
        <width>401</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="priceTableView">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>130</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="reclamationButton">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>330</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>reclamtion</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>-70</y>
        <width>271</width>
        <height>261</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;Notre extras:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Page</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
